quantities/measures/Map.cs

```diff
@@ -1,20 +1,25 @@
+using Quantities.Numerics;
+
 namespace Quantities.Measures;
 
 internal delegate Double Scale(in Double value);
 
 internal sealed class Map
 {
-    public required Scale ToSi { get; init; }
-    public required Scale FromSi { get; init; }
+    private readonly Polynomial conversion;
+    internal Map(Polynomial conversion) => this.conversion = conversion;
     public required IInjector Injector { get; init; }
     public required String Representation { get; init; }
     public required Action<IWriter> Serialize { get; init; }
-
-    public Map With(IInjector injector) => new() {
+    public Map With(IInjector injector) => new(this.conversion) {
         Injector = injector,
-        ToSi = this.ToSi,
-        FromSi = this.FromSi,
         Serialize = this.Serialize,
         Representation = this.Representation
     };
+    public Double Project(in Map other, in Double self)
+    {
+        var siValue = this.conversion.Evaluate(in self);
+        return other.conversion.Inverse(in siValue);
+    }
```

Comment:
Where the polynomial is used to calculate the conversion first from some unit to the si value (´conversion.Evaluate´) and then from that si value to the other unit ('conversion.Invert').

Reply:
My thought process: I find the names "ToSi" instead of "Evaluate" and "FromSi" instead of "Inverse" more explicit. On the other hand, they make no sense in the context of polynomials. So the names are probably fine as they are 😊

Reply:
Additionally, this idea of "ToSi" is considered in the method below. This would be another strong reason to leave things as is 😁